orchestra/proc-macro/src/impl_channels_out.rs

```diff
@@ -81,14 +81,21 @@ pub(crate) fn impl_channels_out_struct(info: &OrchestraInfo) -> Result<proc_macr
 				&mut self,
 				signals_received: usize,
 				message: #message_wrapper,
+				use_priority_channel: bool,
```

Comment:
not a fan of booleans, either introduce and enum or move it to proc-macro generation or rust compile time

Reply:
Replaced with an enum